src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const PRICE_PER_DAY = 40;
+  const LONG_TERM_DISCOUNT = 50;
+  const LONG_TERM = 7;
+  const SHORT_TERM = 3;
+  const SHORT_TERM_DISCOUNT = 20;
+  const BASE_PRICE = days * PRICE_PER_DAY;
```

Comment:
Agree without previous comment, in UPPER_CASE we name only constants. In this case base price just a variable that depends on an argument `days`, so it should be in camelCase